src/screenlock/screenlock.cpp

```diff
@@ -62,6 +65,8 @@ ScreenLock::ScreenLock(TouchScreen *touch, QObject* parent) :
             "display_blanking_policy_ind",
             this,
             SLOT(handleBlankingPolicyChange(QString)));
+
+    m_allowSkippingLockScreen = MGConfItem("/lipstick/skip_lock_screen").value(false).toBool();
```

Comment:
This is user setting, right? Which would not be visible to system service like mce. I'm wondering how much unexpected lockscreen state change dbus ping-pong there might be / whether it would be worthwhile to make also mce aware of such behavioral change to avoid it - i.e. is this the right level to configure this?

Reply:
Not user setting, but device configuration option that for vanilla release is off.